server/server.js

```diff
@@ -1,481 +1,106 @@
-                    { author:"Ian", content:"Nah"}]
-            },
-            {
-            title:"Yea or nah",
-            id:3,
-            author: "Eli",
-            content: "thread content",
-            comments:[
-                { author:"Fran", content:"Nope"},
-                { author:"Greg", content:"Noooooo" },
-                { author:"Hank", content:"Yes please"},
-                { author:"Ian", content:"Nah"}]
-            },
-        {
-                title:"too To TWO",
-                id:4,
-                author: "Jill",
-                comments:[
-                    { author:"Kim", content:"two"},
-                    { author:"Lisa", content:"to" },
-                    { author:"Mary", content:"too"}]
-            
-        }]
-
-    
-        console.log("someone sent a request home");
-        response.send(db.comments);
-    });
-    
-    // Array with comments
-    var threads = 
-        [{ title:"this or that",
-            id:1,
-            author: "Aaron",
-            content: "thread content",
-            comments:[
-                { author:"Brad", content:"this"},
-                { author:"Clark", content:"that" },
-                { author:"Dave", content:"that too"}]
-            },
-            {
-                title:"Yea or nah",
-                id:2,
-                author: "Eli",
-                content: "thread content",
-                comments:[
-                    { author:"Fran", content:"Nope"},
-                    { author:"Greg", content:"Noooooo" },
-                    { author:"Hank", content:"Yes please"},
-                    { author:"Ian", content:"Nah"}]
-            },
-            {
-            title:"Yea or nah",
-            id:3,
-            author: "Eli",
-            content: "thread content",
-            comments:[
-                { author:"Fran", content:"Nope"},
-                { author:"Greg", content:"Noooooo" },
-                { author:"Hank", content:"Yes please"},
-                { author:"Ian", content:"Nah"}]
-            },
-        {
-                title:"too To TWO",
-                id:4,
-                author: "Jill",
-                comments:[
-                    { author:"Kim", content:"two"},
-                    { author:"Lisa", content:"to" },
-                    { author:"Mary", content:"too"}]
-            
-        }]
-
-    
-    
-    app.listen(3000,function(){
-        console.log("listening in port 3000");
-    });
\ No newline at end of file
+const es6Renderer = require("express-es6-template-engine");
+app.engine("html", es6Renderer);
+app.set("views", "templates");
+app.set("view engine", "html");
+
+app.get("/api/threads", function (request, response, next) {
+  console.log("someone sent a request home");
+  response.json(db);
+});
+
+// post to /api/threads is a different route than get to /api/threads
+app.post("/api/threads", function (request, response, next) {
```

Comment:
I am going to leave this to you to implement. You should have the elements in place from the changes I have made here to be able to follow my example to post a thread, pushing it onto the db.